contracts/core/extensions/CoreIssuance.sol

```diff
@@ -198,12 +223,14 @@ contract CoreIssuance is
     /**
      * Exchange components for Set tokens, accepting any owner
      *
-     * @param  _owner        Address to issue tokens to
+     * @param  _owner        Address to use tokens from
```

Comment:
Maybe something a little more descriptive than _owner is necessary now. Like `_componentOwner` and `_setRecipient`

Reply:
+1